gradle/libs.versions.toml

```
@@ -23,6 +23,7 @@ ktlint-plugin = "12.1.1"
 ktlint-version = "1.5.0"
 room = "2.7.0-rc01"
 sqlite = "2.5.0-rc01"
+aboutLibraries = '11.2.3'
```

Please use the latest version:
```suggestion
aboutLibraries = '11.6.3'
```